src/main/java/unicash/model/Model.java

```diff
@@ -82,6 +82,11 @@ public interface Model {
      */
     void addTransaction(Transaction transaction);
 
+    /**
+     * Returns true if UniCash has reached max transaction capacity.
+     */
+    boolean isMax();
```

Comment:
Maybe a more descriptive name like `isFull`?

Reply:
Sure, let me update it